ofl/akayatelivigala/METADATA.pb

```diff
@@ -0,0 +1,22 @@
+name: "Akaya Telivigala"
+designer: "Vaishnavi Murthy, Juan Luis Blanco"
+license: "OFL"
+category: "DISPLAY"
+date_added: "2021-01-14"
+fonts {
+  name: "Akaya Telivigala"
+  style: "normal"
+  weight: 400
+  filename: "AkayaTelivigala-Regular.ttf"
+  post_script_name: "AkayaTelivigala-Regular"
+  full_name: "Akaya Telivigala Regular"
+  copyright: "Copyright 2015 The Akaya Telivigala Project Authors (https://github.com/vaishnavimurthy/Akaya-Telivigala)"
+}
+subsets: "latin"
+subsets: "latin-ext"
+subsets: "menu"
+subsets: "telugu"
+source {
+  repository_url: "https://github.com/vaishnavimurthy/Akaya-Telivigala.git"
+  commit: "db01c66d22057fd8f5e5d433d1ef3b37098ca209"
+}
```

Comment:
When using the packager, you want to include `--no-source`. @davelab6 do we know if this field is supported on the backend yet?

Reply:
Want or have to? What does it do? And shall I change anything now for this PR?

Reply:
apologies but `have` makes more sense. Don't worry about changing anything, I'll take it out myself.